exporter/otlphttpexporter/config.go

```diff
@@ -60,6 +60,10 @@ type Config struct {
 
 	// The encoding to export telemetry (default: "proto")
 	Encoding EncodingType `mapstructure:"encoding"`
+
+	// Experimental: This configuration is at the early stage of development and may change without backward compatibility
+	// until https://github.com/open-telemetry/opentelemetry-collector/issues/8122 is resolved
+	BatcherConfig exporterhelper.BatcherConfig `mapstructure:"batcher"`
```

Comment:
Please don't do this. I am about to remove it. Will add next week I promise the new config.

Reply:
ah i thought this was the new config.... i followed the OTLP exporter's example